sql/parser/sql.y

```diff
@@ -336,7 +338,7 @@ import "strings"
 %token <str>   FALSE FAMILY FETCH FILTER FIRST FLOAT FOLLOWING FOR
 %token <str>   FORCE FOREIGN FORWARD FREEZE FROM FULL FUNCTION FUNCTIONS
 
-%token <str>   GLOBAL GRANT GRANTED GREATEST GROUP GROUPING
+%token <str>   GLOBAL GRANT GRANTED GRANTS GREATEST GROUP GROUPING
```

Comment:
this is now under `unreserved_keywords` and handled in the show statement just like `DATABASES` and `TABLES`.